content/influxdb/cloud-dedicated/admin/sso.md

```diff
@@ -0,0 +1,102 @@
+menu:
+  influxdb_cloud_dedicated:
+    name: Set up and use SSO
+    parent: Administer InfluxDB Cloud
+weight: 106
+---
+
+{{< product-name >}} supports single sign-on (SSO) integrations through the use
+of [Auth0](https://auth0.com) and your identity provider of choice.
+Use SSO to provide users seamless access to your {{< product-name >}} cluster
+with an existing set of credentials. 
+
+{{% cloud %}}
+#### Contact InfluxData sales to enable SSO
+
+SSO is a paid upgrade to your {{< product-name >}} cluster.
+To begin the process of enabling SSO, contact InfluxData Sales:
+
+<a class="btn" href="https://www.influxdata.com/contact-sales/">Contact InfluxData Sales</a>
+{{% /cloud %}}
+
+## SSO authorization flow
+
+With SSO enabled, whenever a user attempts to log into your {{< product-name >}}
+cluster, the following occurs:
+
+1.  InfluxDB sends an authentication request to the InfluxData-managed Auth0 service.
+2.  Auth0 sends the provided credentials to your identity provider.
+3.  Your identity provider grants or denies authorization based on the provided
+    credentials and returns the appropriate response to Auth0.
+4.  Auth0 returns the authorization response to {{< product-name >}} which grants
+    or denies access to the user.
+
+{{< html-diagram/sso-auth-flow >}}
+
+## Set up your identity provider
+
+For information about setting up and configuring your identity provider, refer
+to your identity provider's documentation.
+You can use any identity provider **supported by Auth0**:
+
+- [Social identity providers supported by Auth0 {{< icon "export" >}}](https://auth0.com/docs/authenticate/identity-providers/social-identity-providers)
+- [Enterprise identity providers supported by Auth0 {{< icon "export" >}}](https://auth0.com/docs/authenticate/identity-providers/enterprise-identity-providers)
+- [Legal identity providers supported by Auth0 {{< icon "export" >}}](https://auth0.com/docs/authenticate/identity-providers/enterprise-identity-providers)
+
+## Connect your identity provider to Auth0
+
+To integrate your identity provider with the InfluxData-managed Auth0 service:
+
+1.  **Create a new application or client** in your identity provider to use with
+    Auth0 and your {{< product-name >}} cluster.
+
+2.  **Provide the necessary connection credentials to InfluxData support**.
+    What credentials are needed depends on your identity provider and the
+    protocol you're using. For example:
+
+    | Protocol | Required credentials          |
+    | :------- | :---------------------------- |
+    | **OIDC** | Client secret                 |
+    | **SAML** | Identity provider certificate |
+
+    InfluxData support will provide you with more information about what specific
+    credentials are required.
+
+3. **Add the InfluxData Auth0 connection URL as a valid callback URL** to your
+    identity provider application.
+    InfluxData support will provide you with the connection URL.
```

Comment:
@jlytle-influxdata Is this a standard callback URL or is it specific to each client? If it is standard, should we make it public in the documentation or should we hold it back for support to provide it to customers?

Reply:
> @jlytle-influxdata Is this a standard callback URL or is it specific to each client? If it is standard, should we make it public in the documentation or should we hold it back for support to provide it to customers?

another name for `callback URL` you will see out in the wild is `_post-back URL` - they are the same thing.

for C2, it is standard: https://auth.influxdata.com/login/callback
There is a receiving app in Auth0 called `cloud 2 - prod`, if this changes then the callback URL also could change.